alpine.dockerfile

```diff
@@ -1,24 +1,24 @@
-FROM frolvlad/alpine-glibc:alpine-3.13
+ARG DENO_VERSION=1.13.2
+ARG BIN_IMAGE=denoland/deno:bin-${DENO_VERSION}
+
 
-ENV DENO_VERSION=1.13.2
+FROM ${BIN_IMAGE} AS bin
 
-RUN apk add --virtual .download --no-cache curl \
- && curl -fsSL https://github.com/denoland/deno/releases/download/v${DENO_VERSION}/deno-x86_64-unknown-linux-gnu.zip \
-         --output deno.zip \
- && unzip deno.zip \
- && rm deno.zip \
- && chmod 755 deno \
- && mv deno /bin/deno \
- && apk del .download
+
+FROM frolvlad/alpine-glibc:alpine-3.13
 
 RUN addgroup --gid 1000 deno \
- && adduser --uid 1000 --disabled-password deno --ingroup deno \
- && mkdir /deno-dir/ \
- && chown deno:deno /deno-dir/
+  && adduser --uid 1000 --disabled-password deno --ingroup deno \
```

Comment:
but now the indents don't match 😭

Reply:
I thought it was a mistake (I never saw indents with 1 space only). Do you want me to change to 1 space?